src/ripple/app/tx/impl/CreateOffer.cpp

```diff
@@ -234,6 +234,10 @@ CreateOffer::checkAcceptAsset(ReadView const& view,
             : tecNO_ISSUER;
     }
 
+    if (view.rules().enabled(featureFlowCross) && (issue.account == id))
```

Comment:
Is it worth a comment that this change is coincident with the flow cross amendment, but not itself a flow-cross change?